stytch/b2b/api/organizations_members.py

```diff
@@ -648,27 +648,6 @@ def unlink_retired_email(
         email_address: Optional[str] = None,
         method_options: Optional[UnlinkRetiredEmailRequestOptions] = None,
     ) -> UnlinkRetiredEmailResponse:
-        """Unlinks a retired email address from a Member specified by their `organization_id` and `member_id`. The email address
```

Comment:
Is this just removed from the auto-generation script? This doesn't seem like something we should remove.

Reply:
Yeah I messaged @jcook-stytch about this - he used a local docs branch to do the generation, so that caused this diff. We need to get the docs changes out for retired emails to make sure we don't keep deleting it when generating, but for now I think maybe @etaylormcgregor-stytch can use Jack's branch for docs so that this doesn't get deleted